src/app/commons/ItemListPage/ItemListPage.js

```diff
@@ -17,29 +17,37 @@ const FilterSearchBar = ({filterItems, Button}) => {
     )
 };
 
-
 /**
- * @param title the title (str) of the page
- * @param filterItems the item's name (str) that is included in the filter drop-down list
- * @param onCreateButtonClick the event listener triggered when the green 'Create' button get clicked
- * @param tableHeaders a list of header names (str) in the table
- * @param tableRowGenerator 3 fields (1) list: 'the data list' (2) data: a data generation function that
+ * @callback callback
```

Comment:
This should be incorrect, you should declare the callbacks in another comment section and you also need to declare its parameters.  see: https://jsdoc.app/tags-callback.html

Reply:
@Wally5077 unresolve